tests/github-action-helper.sh

```diff
@@ -109,6 +140,15 @@ deploy_csi_driver_custom_ns() {
   kubectl create -f https://raw.githubusercontent.com/rook/rook/master/deploy/examples/csi/cephfs/pvc.yaml
 }
 
+wait_for_pvc_to_be_bound_state_default() {
+  timeout 100 bash <<-'EOF'
+    until [ $(kubectl get pvc cephfs-pvc-retain -o jsonpath='{.status.phase}') == "Bound" ]; do
+      echo "waiting for the pvc to be in bound state"
+      sleep 1
+    done
+EOF
+}
```

Comment:
```suggestion
EOF
    timeout_command_exit_code
}
```

add above function call to exit if timeout reached